<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>001.Promise第一课</title>
	<script>
		new Promise((resolve,reject)=>{
			resolve('1');
		}).then(res=>{
			console.log(res);
			return new Promise((resolve,reject)=>{
				reject('2');
			})
		}).then(res=>{
			console.log(res);
			return new Promise((resolve,reject)=>{
				resolve('3');
			})
		}).then(res=>{
			console.log(res);
		}).catch(err=>{
			console.log('出错了');
		})

	</script>
</head>
<body>
	<p>
		在Promise解决回调炼狱时，若前面有错误（发送reject信号）那么下面的所有请求将不会再执行而是直接进入异常
	</p>
</body>
</html>